next-types.d.ts

```diff
@@ -605,3 +605,15 @@ interface DbWorkspacesReposStats {
     activity_ratio: number;
   };
 }
+
+interface ContributorLottoFactor {
+  contributor: string;
+  count: number;
+  percent_of_total: number;
+  lotto_factor: "very-high" | "high" | "moderate" | "low";
+}
+
+interface RepositoryLottoFactor {
+  all_contribs: ContributorLottoFactor[];
+  all_lotto_factor: "very-high" | "high" | "moderate" | "low";
+}
```

Comment:
Refactor (blocking): No need to duplicate the type.

```suggestion
type LottoFactor = "very-high" | "high" | "moderate" | "low"

interface ContributorLottoFactor {
  contributor: string;
  count: number;
  percent_of_total: number;
  lotto_factor: LottoFactor;
}

interface RepositoryLottoFactor {
  all_contribs: ContributorLottoFactor[];
  all_lotto_factor: LottoFactor;
}
```